server/planning/assignments/assignments.py

```diff
@@ -829,6 +829,18 @@ def is_assignment_being_activated(self, updates, original):
         return original.get('assigned_to').get('state') == ASSIGNMENT_WORKFLOW_STATE.DRAFT and\
             updates.get('assigned_to', {}).get('state') == ASSIGNMENT_WORKFLOW_STATE.ASSIGNED
 
+    def is_text_assignment(self, assignment):
+        if assignment.get('planning', {}).get('g2_content_type') == 'text':
+            return True
+
+        content_types = get_resource_service('vocabularies').find_one(req=None, _id='g2_content_type')
+        if content_types:
+            text_types = [t.get('qcode') for t in (content_types.get('items') or []) if
+                          t.get('qcode') == 'text' or t.get('content item type') == 'text']
```

Comment:
And here as well, use `content item type` instead of `qcode`.

@mdhaman This might mean we will require an upgrade script or to manually add `content item type` to our CV!